### ISSUE TITLE
Investigate openshift ansible 3.9

### ISSUE THREAD

**Issue by cmoulliard** [snowdrop/cloud-native-infra#3]
For 3.7 and below, you need to do some manual preparation steps and
then the playbook you want to run is:
```
openshift-ansible/playbooks/byo/config.yml
```

For 3.9 (when the rpms will be ready)
```
Prerequisites: https://docs.openshift.org/latest/install_config/install/prerequisites.html
Host prep: https://docs.openshift.org/latest/install_config/install/host_preparation.html
```
I think some of the items in those pages are already done on atomic
host (such as installing docker).

Interesting project : https://github.com/michaelgugino/openshift-stuff/tree/master/centos
